### DISCUSSION TITLE
How can Hypernetworks be added in the future?

### DISCUSSION THREAD

**Discussion by Lling00** [comfyanonymous/ComfyUI#529] [General]
Is it possible to add Hypernetworks to ComfyUI? I have a lot of Hypernetworks files on my local machine, but I can't find a way to use them in ComfyUI. It would be great if there is a Hypernetworks node available in ComfyUI.

**Comment by omar92** [comment]
try use lora i dont see any one creata Hypernetworks  anymore

**Comment by omar92** [comment]
mostly you will find loras do same things you had

**Reply by Lling00** [reply]
However, there are times when Hypernetworks and lora are needed together

**Comment by comfyanonymous** [comment]
I have added support for linear hypernetworks, put them in the models/hypernetworks folder and use the new HypernetworkLoader node in _for_testing to load them.

Check your console, if it prints that your hypernetwork isn't supported let me know and I might write the code to support it.

**Reply by Lling00** [reply]
Without using HypernetworkLoader.
![image](https://github.com/comfyanonymous/ComfyUI/assets/48402714/255c6db8-f409-4cc2-89d3-886e3d973d52)

After using HypernetworkLoader, the generated images look strange. I'm not sure if there is a problem with my node connections.
![image](https://github.com/comfyanonymous/ComfyUI/assets/48402714/b33ab6c5-77c8-4467-9eb1-dbd9ed111dd3)
![image](https://github.com/comfyanonymous/ComfyUI/assets/48402714/9bb1654c-b72b-4a4b-aa84-89be24ca5e05)

**Reply by comfyanonymous** [reply]
you can try lowering (worst quality, low quality) to 1.2

Can you give me a link to that hypernetwork?

**Reply by Lling00** [reply]
https://pan.baidu.com/s/1RcpEmJKc3VMAgta_oyPq9w 
e9lq 
This is a super network file
